docs/1.12/03-Tutorials2/06-Deploy-Prisma-Servers/06-Zeit-Now-and-Google-Cloud-SQL.md

```diff
@@ -0,0 +1,158 @@
+
+---
+
+* You can learn the additional [command line flags](https://cloud.google.com/sdk/gcloud/reference/beta/sql/instances/create)
+* The steps for creating a Postgres database instance are virtually identical.
+
+
+### 1.2. Create SQL User
+
+Create a user called `prisma` with a password of `my-secret`.
+
+```console
+❯ gcloud beta sql users create prisma --instance=prisma --password=my-secret
+```
+
+Your MySQL Instance is now up and running.
+
+Note the IP address of the instance, your username and password.
+
+## 2. Deploying a Prisma server to Zeit Now
+
+Deployment to Now consists of 3 files; a Dockerfile, prisma configuration and Now deployment configuration.
+
+### 2.1 Now Secrets
+
+First we will add secrets to Now to store the sql password from earlier and the Prisma API password.
+
+```console
+❯ now secret add sql-password my-secret
+❯ now secret add prisma-management-api-secret so-secret
+```
+
+*  Learn more about Now secrets: https://zeit.co/docs/getting-started/secrets
+
+### 2.2 Prisma Configuration
+
+Create a file `config.yml`. This contains the Prisma Server configuration.
+
+```file
+managementApiSecret: PRISMA_MANAGEMENT_API_SECRET
+port: 4466
+databases:
+    default:
+        connector: mysql
+        host: SQL_HOST
+        port: 3306
+        user: prisma
+        password: SQL_PASSWORD
+        migrations: true
+        active: true
+```
+
+* SQL_HOST, SQL_PASSWORD and PRISMA_MANAGEMENT_API_SECRET will be replaced with Now build-time environment variables.
+
+### 2.2 Now Deployment Configuration
+
+Create `now.json`. This file contains our build environment variables and references to the secrets.
+
+Change `xx.xx.xx.xx` to the Cloud SQL IP address returned in 1.1.
+
+```file
+{
+  "name": "prisma-now",
+  "type": "docker",
+  "build": {
+    "env": {
+      "SQL_HOST": "xx.xx.xx.xx",
+      "SQL_PASSWORD": "@sql-password",
+      "PRISMA_MANAGEMENT_API_SECRET": "@prisma-management-api-secret"
+    }
+  },
+}
+```
+
+* Learn more about environment variables: https://zeit.co/docs/features/build-env-and-secrets and more about `now.json`: https://zeit.co/docs/features/configuration.
+* To take advantage of Zeit's automatic scaling: https://zeit.co/docs/getting-started/scaling
+
+### 2.3 Dockerfile
+
+Create a file called `Dockerfile`. This tells Now how to build and configure the Prisma server.
+
+```file
+FROM prismagraphql/prisma:1.13-beta
+ARG SQL_HOST
+ARG SQL_PASSWORD
+ARG PRISMA_MANAGEMENT_API_SECRET
+ARG PRISMA_CONFIG_PATH
+ENV PRISMA_CONFIG_PATH prisma.yml
+COPY config.yml prisma.yml
+RUN sed -i s/SQL_HOST/$SQL_HOST/g prisma.yml
+RUN sed -i s/SQL_PASSWORD/$SQL_PASSWORD/g prisma.yml
+RUN sed -i s/PRISMA_MANAGEMENT_API_SECRET/$PRISMA_MANAGEMENT_API_SECRET/g prisma.yml
+EXPOSE 4466
+```
+
+* Edit `Dockerfile` with the version of prisma you wish to deploy from [dockerhub](https://hub.docker.com/r/prismagraphql/prisma/tags/)
+
+### 2.4 Deploy
+
+Then you are ready to deploy.
+
+```console
+❯ now
```

Comment:
remove the `❯` character in front of the bash command which allows people to directly copy the code from the code block and paste it into the terminal